repair/featurize/__init__.py

```diff
@@ -3,11 +3,10 @@
 from .initfeat import InitFeaturizer
 from .initsimfeat import InitSimFeaturizer
 from .freqfeat import FreqFeaturizer
-from .occurfeat import OccurFeaturizer
 from .constraintfeat import ConstraintFeat
 from .langmodel import LangModelFeat
 from .initattfeat import InitAttFeaturizer
 from .occurattrfeat import OccurAttrFeaturizer
 
 __all__ = ['FeaturizedDataset', 'Featurizer', 'InitFeaturizer', 'InitSimFeaturizer', 'FreqFeaturizer',
```

Comment:
We can remove `InitFeaturizer` here and the import above as well.

Reply:
Ok, removed the whole class.

Reply:
Also sorted imports